src/api/resources/delegates.js

```diff
@@ -33,5 +33,16 @@ export default class DelegatesResource extends APIResource {
 				offset: 101,
```

Comment:
API spec says `sort` instead of `orderBy`, but that might be the default on the API anyway?